[2021-06-24 16:20:23,764] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [queued]>
[2021-06-24 16:20:23,775] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [queued]>
[2021-06-24 16:20:23,775] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-24 16:20:23,775] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-24 16:20:23,775] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-24 16:20:23,790] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-01-02T00:00:00+00:00
[2021-06-24 16:20:23,793] {standard_task_runner.py:52} INFO - Started process 1924 to run task
[2021-06-24 16:20:23,802] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-01-02T00:00:00+00:00', '--job-id', '43', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpuesprod8', '--error-file', '/tmp/tmpklw76bun']
[2021-06-24 16:20:23,803] {standard_task_runner.py:77} INFO - Job 43: Subtask task_1
[2021-06-24 16:20:23,877] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [running]> on host airflowvm
[2021-06-24 16:20:23,967] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-01-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-02T00:00:00+00:00
[2021-06-24 16:20:23,967] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-24 16:20:23,968] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-24 16:20:23,974] {subprocess.py:75} INFO - Output:
[2021-06-24 16:20:26,977] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-24 16:20:27,001] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200102T000000, start_date=20210624T162023, end_date=20210624T162027
[2021-06-24 16:20:27,055] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-06-24 16:20:27,091] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-06-24 17:03:24,339] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [queued]>
[2021-06-24 17:03:24,468] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [queued]>
[2021-06-24 17:03:24,474] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-24 17:03:24,475] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-24 17:03:24,475] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-24 17:03:24,621] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-01-02T00:00:00+00:00
[2021-06-24 17:03:24,636] {standard_task_runner.py:52} INFO - Started process 8560 to run task
[2021-06-24 17:03:24,841] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-01-02T00:00:00+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp8r8atrbr', '--error-file', '/tmp/tmppee6cltu']
[2021-06-24 17:03:24,856] {standard_task_runner.py:77} INFO - Job 3: Subtask task_1
[2021-06-24 17:03:25,540] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-01-02T00:00:00+00:00 [running]> on host airflowvm
[2021-06-24 17:03:25,980] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-01-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-02T00:00:00+00:00
[2021-06-24 17:03:25,988] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-24 17:03:26,029] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-24 17:03:26,146] {subprocess.py:75} INFO - Output:
[2021-06-24 17:03:29,252] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-24 17:03:29,763] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200102T000000, start_date=20210624T170324, end_date=20210624T170329
[2021-06-24 17:03:30,601] {local_task_job.py:196} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-06-24 17:03:30,847] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 8560
[2021-06-24 17:03:33,175] {taskinstance.py:1264} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-06-24 17:03:33,177] {subprocess.py:89} INFO - Sending SIGTERM signal to process group
[2021-06-24 17:03:33,329] {process_utils.py:66} INFO - Process psutil.Process(pid=8560, status='terminated', exitcode=1, started='17:03:23') (8560) terminated with exit code 1
